changelog.txt

```diff
@@ -1,54 +1,69 @@
-4.42.1
+4.43.0
 ==========
 
-### Bug fixes
+### New Features
+**Added Intune Mobile App Management (MAM) support for Android**. See [3185](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3185).  
+**MSAL.NET Cache Extensions now protects plaintext cache files with owner only read/write permissions**.See [3186](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3186), [169](https://github.com/AzureAD/microsoft-authentication-extensions-for-dotnet/issues/169).  
+
+### Bug Fixes
+**Client capabilities flags are correctly passed to Android Broker**. See [3203](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3203).  
+**Fixed `WithAccount(result.Account)` to work when using WAM**. See [3121](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3121).   
+**Improved token cache filtering logic**. See [3178](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3178), and [3233](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3233).  
+**Fixed an error in creating UWP package for Microsoft Store upload**.  See [3184](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3184), [3239](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3239).  
```

Comment:
This was fixed in 4.42.1

Reply:
I thought it didn't work, so we had to do https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3239?

Reply:
that's correct do we need to mention #3184 on both releases?

Reply:
I think yea, since it's the original issue and it's linked to some other duplicate issues.